lib/Braintree/WebhookTestingGateway.php

```diff
@@ -477,6 +489,51 @@ private static function _subscriptionChargedUnsuccessfullySampleXml($id)
         ";
     }
 
+    private static function _subscriptionExpired($id) {
```

Comment:
```suggestion
    private static function _subscriptionExpiredSampleXml($id) {
```
Small suggestion -- let's add `SampleXml` to the end of these new method names for consistency with the rest of the class.